sky/resources.py

```diff
@@ -114,7 +116,10 @@ def __repr__(self) -> str:
 
         image_id = ''
         if self.image_id is not None:
-            image_id = f', image_id={self.image_id!r}'
+            if None in self.image_id:
+                image_id = f', image_id={self.image_id[None]}'
+            else:
+                image_id = f', image_id={self.image_id!r}'
```

Comment:
If there are many image ids, `sky status` becomes a little crowded:
```
NAME  LAUNCHED        RESOURCES                                                                  STATUS  AUTOSTOP  COMMAND
clus  a few secs ago  1x AWS(m6i.2xlarge, image_id={'us-east-1': 'ami-0fb70a87623506e3e', 'u...  UP      -         sky launch -c clus test.y...
```

I'm trying to think of a better way to do this but not sure what's the best:
1. List only regions/zones: `image_id={us-east-1,us-west-1}`. Pros - gives a hint which images are specified. Cons - can be confusing, no details about the specific image used.
2. Show the count of images specified: `image_id={5 images}`. Pros - concise. Cons - no details about region/images
3. Do not show image_id in sky status. Pros - concise. Cons - No details.
4. Keep as it is. Pros - detailed. Cons - too long

Reply:
On thinking more, this field is shown only if image_id is specified. And if a user is specifying an image_id, they would want to know which image is being used for a task. So choosing option 4 (keep as is) might be okay.

Reply:
^that's an excellent idea! I like it